tests/test_rabfe.py

```diff
@@ -287,6 +310,72 @@ def test_predict_relative_binding_model(self):
             self.assertEqual(len([x for x in created_files if x.endswith(".npy")]), 2)
             self.assertEqual(len([x for x in created_files if x.endswith(".npz")]), 6)
 
+    def test_predict_relative_conversion_model(self):
+        """Just to verify that we can handle the most basic decoupling RABFE prediction"""
```

Comment:
Preference for more descriptive test summary.

(Asserts that `RelativeConversionModel`'s `predict` function runs, is deterministic, and creates the expected number of pdb, npy, npz files)

Reply:
This was copy and pasted from the decoupling test - but I agree that your docstring is better.